integrationtests/helpers/accounts.py

```diff
@@ -48,19 +48,30 @@ def _clear_minio():
                 raise err
 
 
-def login(browser: Browser, email: str, password: str) -> None:
+def login(
```

Comment:
I don't think these helpers should change. Please move your locale-enhanced functions into `test_i18n.py` instead.

It's a judgment call. I know i18n is a cross-cutting concern; but that's true of almost _everything_ we integration test. Each test is more legible when _its_ cross-cutting concern doesn't impact the helpers all the _other_ test files use.